var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -432,9 +460,9 @@ def configure_options(self):
                 ('scotch', 'ptscotch', True, True),
                 ('suite-sparse:umfpack,klu,cholmod,btf,ccolamd,colamd,camd,amd, \
                 suitesparseconfig,spqr', 'suitesparse', True, True),
-                ('hdf5:hl,fortran', 'hdf5', True, True),
                 'zlib',
                 'mumps',
+                ('hdf5'+hdf5libs, 'hdf5', True, True),
```

Comment:
suggest moving this line to the prior location [to have a better diff]

Reply:
Thanks for catching that. This has now been fixed.